src/mjolnir/elevationbuilder.cc

```diff
@@ -56,10 +56,14 @@ std::vector<int8_t> encode_edge_elevation(const std::unique_ptr<valhalla::skadi:
   bool error = false;
   std::vector<int8_t> encoded = encode_elevation(heights, error);
   if (error) {
-    LOG_INFO("edge elevation wayid= " + std::to_string(wayid));
-    for (const auto h : heights) {
-      LOG_INFO("   " + std::to_string(h));
+    double diff = 0;
+    for (size_t i = 1; i <= heights.size(); i++) {
+      auto d = std::abs(heights[i] - heights[i - 1]);
+      diff = d < diff ? diff : d;
+      LOG_DEBUG("  " + std::to_string(heights[i - 1]));
     }
+    LOG_WARN("edge elevation wayid = " + std::to_string(wayid) + " exceeds difference with " +
```

Comment:
only this line will be printed by default. I have no feeling if this will happen a lot. it'd be nice to have the way id in the logs when this happens, since encoded elevations can be really really off for those ways. good in case issues roll in regarding that. 

I'll try one planet with this and see how noisy this is now.